Source/iOS/OIDAuthorizationUICoordinatorIOS.m

```diff
@@ -87,37 +87,38 @@ - (BOOL)presentAuthorizationRequest:(OIDAuthorizationRequest *)request
   _session = session;
   BOOL openedSafari = NO;
   NSURL *requestURL = [request authorizationRequestURL];
-
-  if (@available(iOS 11.0, *)) {
-    NSString *redirectScheme = request.redirectURL.scheme;
-    SFAuthenticationSession* authenticationVC =
+    if (@available(iOS 11.0, *)) {
+        __weak __typeof(self) weakSelf = self;
+        NSString *redirectScheme = request.redirectURL.scheme;
+        SFAuthenticationSession* authenticationVC =
         [[SFAuthenticationSession alloc] initWithURL:requestURL
                                    callbackURLScheme:redirectScheme
                                    completionHandler:^(NSURL * _Nullable callbackURL,
                                                        NSError * _Nullable error) {
-      _authenticationVC = nil;
-      if (callbackURL) {
-        [_session resumeAuthorizationFlowWithURL:callbackURL];
-      } else {
-        NSError *safariError =
-            [OIDErrorUtilities errorWithCode:OIDErrorCodeUserCanceledAuthorizationFlow
-                             underlyingError:error
-                                 description:nil];
-        [_session failAuthorizationFlowWithError:safariError];
-      }
-    }];
-    _authenticationVC = authenticationVC;
-    openedSafari = [authenticationVC start];
-  } else if (@available(iOS 9.0, *)) {
-      SFSafariViewController *safariVC =
-          [[[self class] safariViewControllerFactory] safariViewControllerWithURL:requestURL];
-      safariVC.delegate = self;
-      _safariVC = safariVC;
-      [_presentingViewController presentViewController:safariVC animated:YES completion:nil];
-      openedSafari = YES;
-  } else {
-    openedSafari = [[UIApplication sharedApplication] openURL:requestURL];
-  }
+                                       OIDAuthorizationUICoordinatorIOS *strongSelf = weakSelf;
```

Comment:
The change to the indentation level here makes several of these lines exceed the 100 column limit. We should probably setup a clang-format script so that people don't have to be bothered to make these sort of changes. Sorry :/